client/app/components/DirectoryContent.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import FileRow from './FileRow';
+import DirectoryRow from './DirectoryRow';
+
+const DirectoryContent = (props) => {
+  let files = [];
+  let directories = [];
```

Comment:
Done to, no idea why github ignores changes.
https://github.com/sudodoki/offline-markdown-previewer/pull/15/commits/4169e467cd3800c941889f13db32ec563f9cc1f1